### PR TITLE
qc: add page

### PR DESCRIPTION
**Pull request by pbek**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** 0.5.1

### CODE REVIEW

**Comment by pbek** [comment]
> pages/common/qc.md:4: TLDR004 Command descriptions should end in a period

I don't understand this one, because the line already ends with a period.

File: pages/common/qc.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# qc
+
+> CLI snippet manager to execute command snippets stored in notes in [QOwnNotes](https://www.qownnotes.org/).
+> More information: <https://www.qownnotes.org/getting-started/command-line-snippet-manager.html>.
+
+- Configure the snippet manager, e.g. to set the security token from QOwnNotes:
+
+`qc configure`
+
+- Search for snippets stored in your `Commands.md`:
```
Comment (kbdharun) [inline pages/common/qc.md:10]: Maybe we can update this description to match with your docs?

```suggestion
- Search and print command snippets:
```

Alternatively, we could reword it to:

```suggestion
- Search and print command snippets stored in the `Commands.md` file:
```

We generally avoid using terms like "your", etc in the descriptions. Feel free to reword my suggestions.

**Review by kbdharun — COMMENTED** [review]
Welcome to tldr and thanks for your contribution. I have some minor suggestions for this page (after which the PR is GTG).

_I have heard of QOwnNotes before it's a cool project; keep up the great work ._ 😄

----

> I don't understand this one, because the line already ends with a period.

Yeah, not sure what triggered the output.

File: pages/common/qc.md (hunk 1 above)
Comment (kbdharun) [reply pages/common/qc.md:10]: Is the filename `Commands.md` hard coded? Or does it allow specifying different file names too? In that case, can you update the example for the same?

Comment (pbek) [reply pages/common/qc.md:10]: `Commands.md` is just the default. The user can change that.

**Comment by pbek** [comment]
Thanks a lot for your quick feedback. I tried to resolve the issues. How is looking now?

> I have heard of QOwnNotes before it's a cool project; keep up the great work . 😄

Oh, thanks a lot! 😄🥳🎉 I love tldr, thank you for managing all that documentation!

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages/common/qc.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# qc
+
+> CLI snippet manager to execute command snippets stored in notes in QOwnNotes.
```
Comment (vitorhcl) [inline pages/common/qc.md:3]:
```suggestion
> CLI snippet manager to execute command snippets stored in notes in QOwnNotes.
> See also: `qownnotes`.
```
What do you think of suggesting the page I've added?

**Review by vitorhcl — APPROVED** [review]
Looks good to me, thanks for your contribution! I have a minor suggestion

File: pages/common/qc.md (hunk 2 above)
Comment (pbek) [reply pages/common/qc.md:3]: Hah, sure! I just now realized qownnotes has an entry 😄 thank you

File: pages/common/qc.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
```
Comment (vitorhcl) [inline pages/common/qc.md:3]:
```suggestion
> Manage and execute command snippets stored in QOwnNotes notes.
```
What do you think of simplifying the description a littpe bit?

Comment (pbek) [reply pages/common/qc.md:3]: LGTM, thank you!

**Review by kbdharun — APPROVED** [review]

**Comment by pbek** [comment]
Thank you very much!
